libs/domain/cart/entry/src/entry.component.ts

```diff
@@ -120,7 +121,7 @@ export class CartEntryComponent
         <oryx-icon-button
           size=${Size.Md}
           @click=${this.removeEntry}
-          ?disabled=${this.$isBusy()}
+          ?disabled=${this.$isBusy() && this.loading}
```

Comment:
let's remove fix for entry in this PR (or alternatively solve it with CartService.isBusy with qualifier for entry